lib/sidekiq/fetch.rb

```diff
@@ -31,7 +31,7 @@ def initialize(options)
       @queues = @options[:queues].map { |q| "queue:#{q}" }
       if @strictly_ordered_queues
         @queues.uniq!
-        @queues << TIMEOUT
+        @queues << {timeout: TIMEOUT}
```

Comment:
what is the significance of this change?

Reply:
passing the timeout as an int is deprecated

https://github.com/redis/redis-rb/blob/0afcf1cad6f4fc74213f2b4c7a20819581c36897/lib/redis/commands/lists.rb#L253-L262